cmake/SpectreSetupPythonPackage.cmake

```diff
@@ -44,6 +44,17 @@ file(COPY "${CMAKE_BINARY_DIR}/tmp/spectre"
   FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
     GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 
+# Also link the Python interpreter to bin/python-spectre as an easy way to jump
+# into a REPL or run a script that uses pybindings
```

Comment:
Can you spell out what `REPL` is? I have no idea what this means and searching online isn't helpful...

Reply:
REPL means "read-evaluate-print loop". I replaced it with "Python shell". It's the interactive session you get when you just type `python` on the command line.